.github/workflows/update-self-version-reference.yml

```
@@ -51,8 +51,13 @@ jobs:
             exit 1
           fi
           echo "LATEST_VERSION=$LATEST_VERSION" >> "$GITHUB_ENV"
-          echo "HAS_CHANGES=$(git status --porcelain -z | tr '\0' ' ')" >> "$GITHUB_ENV"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)